src/org-manager/containers/new-pbas/new-pbas.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { Observable, Subscription } from 'rxjs';
+import { filter, take, takeWhile } from 'rxjs/operators';
+
+import * as fromRoot from '../../../app/store';
+import { OrganisationVM } from '../../models/organisation';
+import * as fromStore from '../../store';
+
+@Component({
+  selector: 'app-new-pbas',
+  templateUrl: './new-pbas.component.html'
+})
+export class NewPBAsComponent implements OnInit, OnDestroy {
+
+  public orgs$: Observable<OrganisationVM>;
+  public organisationId: string;
+  public isActiveOrg = false;
+
+  private getAllLoadedSubscription: Subscription;
+
+  constructor(
+    private readonly store: Store<fromStore.OrganisationRootState>
+  ) { }
+
+  public ngOnInit(): void {
+    this.getAllLoadedSubscription = this.store.pipe(select(fromStore.getAllLoaded)).pipe(takeWhile(loaded => !loaded)).subscribe(loaded => {
+      if (!loaded) {
+        this.store.dispatch(new fromStore.LoadActiveOrganisation());
+        this.store.dispatch(new fromStore.LoadPendingOrganisations());
+      }
+    });
+
+    this.orgs$ = this.store.pipe(select(fromStore.getActiveAndPending));
+    this.orgs$.pipe(
+      filter(value => value !== undefined),
+      take(1)
+    ).subscribe(({ organisationId, pbaNumber, isAccLoaded, status }) => {
+      this.organisationId = organisationId;
+      if (status === 'ACTIVE') {
```

Comment:
all is good only a few minor things,
line 40: if it s one statement ternary operator may be a good fit for this case
line 48-53: organizing and aligning may be better

Reply:
Removed the statement as it was setting a property that is no longer used by the template. Also tidied up the brackets / alignment.